src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/ManageApprenticesOrchestrator.cs

```diff
@@ -188,10 +189,13 @@ public async Task<ExtendedApprenticeshipViewModel> GetApprenticeshipForEdit(long
 
             var apprenticeship = _apprenticeshipMapper.MapApprenticeship(data.Apprenticeship, commitmentData.Commitment);
 
+            var includeFrameworks = commitmentData.Commitment.ApprenticeshipEmployerTypeOnApproval !=
+                                    ApprenticeshipEmployerType.NonLevy;
+
```

Comment:
I know this is not part of this PR, but surely the logic on showing frameworks is also dependent on the IsTransferFunded status (was this missed out  previously)

Reply:
Handled differently